packages/common/src/types/command/PartialTargetDescriptor.types.ts

```diff
@@ -189,6 +189,8 @@ export type ScopeType =
   | CustomRegexScopeType
   | OneOfScopeType;
 
+export type ScopeTypeType = ScopeType["type"];
```

Comment:
Do we still need this if we migrate the places I suggest to `SimpleScopeTypeType`?